### PR TITLE
Remove "framework" mode, as it's no longer in use.

### PR DESCRIPTION
**Pull request by techtangents**

### CODE REVIEW

File: modules/server/src/main/ts/bedrock/core/Settings.ts [hunk 1]
```diff
@@ -8,38 +8,23 @@ export interface BedrockSettings {
...
-
-export interface BedrockManualSettings extends BedrockSettings {
   readonly config: string;
   readonly coverage: string[];
   readonly customRoutes: string;
   readonly polyfills: string[];
   readonly testfiles: string[];
 }
 
+export type BedrockManualSettings = BedrockSettings;
```
Comment (lnewson) [inline modules/server/src/main/ts/bedrock/core/Settings.ts:18]: I'd like for this to remain the same as `Clis`, so can we either revert this change or move all the options from `forManual` to `commonOptions`?

**Review by lnewson — APPROVED** [review]

File: modules/server/src/main/ts/bedrock/core/Settings.ts (hunk 1 above)
Comment (techtangents) [reply modules/server/src/main/ts/bedrock/core/Settings.ts:20]: I've done the latter. I also removed the weird stopOnFailure__hidden setting. 

Can you please double-check the changes?

**Review by ashfordneil — APPROVED** [review]

**Review by spocke — APPROVED** [review]
